### ISSUE TITLE
Child bridge crashing since upgrade: TypeError: Cannot read properties of null (reading 'temperature')

### ISSUE THREAD

**Issue by jfmach** [hjdhjd/homebridge-unifi-protect#905]
**Describe The Problem:**
After upgrading the Unifi-Protect plugin to v6.0.1, the child bridge keeps crashing/restarting

**To Reproduce:**
No steps really, I just upgraded and started experiencing the error. Restarting homebridge does not help.

**Logs:**

```
[27/03/2023, 16:52:15] [homebridge-unifi-protect] Launched child bridge with PID 8568
[27/03/2023, 16:52:16] Registering platform 'homebridge-unifi-protect.UniFi Protect'
[27/03/2023, 16:52:16] [homebridge-unifi-protect] Loaded homebridge-unifi-protect v6.0.1 child bridge successfully
[27/03/2023, 16:52:16] Loaded 11 cached accessories from cachedAccessories.0EF8354E1149.
[27/03/2023, 16:52:17] Homebridge v1.6.0 (HAP v0.11.0) (homebridge-unifi-protect) is running on port 44494.
[27/03/2023, 16:52:17] [homebridge-unifi-protect] Connected to the UniFi Protect API at a.b.c.d.
TypeError: Cannot read properties of null (reading 'temperature')
    at ProtectNvrSystemInfo.getCpuTemp (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:209:47)
    at ProtectNvrSystemInfo.configureTemperatureSensor (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:201:94)
    at ProtectNvrSystemInfo.updateDevice (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:141:13)
    at ProtectNvrSystemInfo.configureAccessory (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:102:33)
    at new ProtectNvrSystemInfo (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:28:10)
    at ProtectNvr.ufpStartupHandler (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr.ts:184:23)
    at Object.onceWrapper (node:events:628:26)
    at ProtectApi.emit (node:events:513:28)
    at ProtectApi.bootstrapController (file:///usr/local/lib/node_modules/homebridge-unifi-protect/node_modules/unifi-protect/src/protect-api.ts:256:10)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
[27/03/2023, 16:52:18] [homebridge-unifi-protect] Child bridge process ended
[27/03/2023, 16:52:18] [homebridge-unifi-protect] Process Ended. Code: 1, Signal: null
[27/03/2023, 16:52:25] [homebridge-unifi-protect] Restarting Process...

[27/03/2023, 16:52:26] [homebridge-unifi-protect] Launched child bridge with PID 8579
[27/03/2023, 16:52:26] Registering platform 'homebridge-unifi-protect.UniFi Protect'
[27/03/2023, 16:52:26] [homebridge-unifi-protect] Loaded homebridge-unifi-protect v6.0.1 child bridge successfully
```

**Homebridge Configuration:**

```json
        {
            "name": "Config",
            "port": 8581,
            "auth": "form",
            "theme": "auto",
            "tempUnits": "f",
            "lang": "auto",
            "platform": "config"
        },
        {
            "controllers": [
                {
                    "address": "removed",
                    "password": "removed",
                    "username": "removed"
                }
            ],
            "options": [
                "Enable.Nvr.SystemInfo"
            ],
            "_bridge": {
                "username": "removed",
                "port": 44494
            },
            "platform": "UniFi Protect"
        },
```

**Screenshots:**
N/A

**Environment:**

* **Homebridge Version:**  1.6.0
* **Node Version:**  v18.14.2
* **Homebridge-UniFi-Protect Plugin Version:** 6.0.1
* **UniFi Protect Version:**  2.2.11
* **UniFi Protect Controller Type:**  UCKgen2
* **UniFi Protect Controller Firmware Version:**  2.4.8
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:** N/A
* **Operating System and OS Version:**  Debian 10.11

**Comment by hjdhjd** [comment]
Except it shouldn't do that. 😄 Will add it to the queue...thanks for finding this. Sorry for the trouble.

**Comment by jfmach** [comment]
Thanks for this great plugin and for your support :)

**Comment by jfmach** [comment]
@hjdhjd confirmed. I put the option back in and it is crashing again.
That is correct I have a UCK-G2-PLUS.
Let me know if you want me to capture additional details.

**Comment by jfmach** [comment]
Well, except something did change: it crashed four times, and now seems to be staying up.

I should point out that I have been experiencing performance issues with my cloud key lately, and will be replacing it with an NVR. Maybe this is triggered if the cloud key is not responsive when the bridge starts?

Here is the latest log:
```
[27/03/2023, 17:28:40] Registering platform 'homebridge-unifi-protect.UniFi Protect'
[27/03/2023, 17:28:40] [homebridge-unifi-protect] Loaded homebridge-unifi-protect v6.0.1 child bridge successfully
[27/03/2023, 17:28:45] Homebridge v1.6.0 (HAP v0.11.0) (homebridge-unifi-protect) is running on port 44494.
[27/03/2023, 17:28:46] [homebridge-unifi-protect] Connected to the UniFi Protect API at a.b.c.d.
TypeError: Cannot read properties of null (reading 'temperature')
    at ProtectNvrSystemInfo.getCpuTemp (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:209:47)
    at ProtectNvrSystemInfo.configureTemperatureSensor (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:201:94)
    at ProtectNvrSystemInfo.updateDevice (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:141:13)
    at ProtectNvrSystemInfo.configureAccessory (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:102:33)
    at new ProtectNvrSystemInfo (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:28:10)
    at ProtectNvr.ufpStartupHandler (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr.ts:184:23)
    at Object.onceWrapper (node:events:628:26)
    at ProtectApi.emit (node:events:513:28)
    at ProtectApi.bootstrapController (file:///usr/local/lib/node_modules/homebridge-unifi-protect/node_modules/unifi-protect/src/protect-api.ts:256:10)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
[27/03/2023, 17:28:47] [homebridge-unifi-protect] Child bridge process ended
[27/03/2023, 17:28:47] [homebridge-unifi-protect] Process Ended. Code: 1, Signal: null

[27/03/2023, 17:28:54] [homebridge-unifi-protect] Restarting Process...
[27/03/2023, 17:28:55] [homebridge-unifi-protect] Launched child bridge with PID 10043
[27/03/2023, 17:28:56] Registering platform 'homebridge-unifi-protect.UniFi Protect'
[27/03/2023, 17:28:56] [homebridge-unifi-protect] Loaded homebridge-unifi-protect v6.0.1 child bridge successfully
[27/03/2023, 17:28:56] Loaded 11 cached accessories from cachedAccessories.0EF8354E1149.
[27/03/2023, 17:28:57] Homebridge v1.6.0 (HAP v0.11.0) (homebridge-unifi-protect) is running on port 44494.
[27/03/2023, 17:28:57] [homebridge-unifi-protect] Connected to the UniFi Protect API at a.b.c.d.
TypeError: Cannot read properties of null (reading 'temperature')
    at ProtectNvrSystemInfo.getCpuTemp (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:209:47)
    at ProtectNvrSystemInfo.configureTemperatureSensor (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:201:94)
    at ProtectNvrSystemInfo.updateDevice (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:141:13)
    at ProtectNvrSystemInfo.configureAccessory (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:102:33)
    at new ProtectNvrSystemInfo (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:28:10)
    at ProtectNvr.ufpStartupHandler (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr.ts:184:23)
    at Object.onceWrapper (node:events:628:26)
    at ProtectApi.emit (node:events:513:28)
    at ProtectApi.bootstrapController (file:///usr/local/lib/node_modules/homebridge-unifi-protect/node_modules/unifi-protect/src/protect-api.ts:256:10)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
[27/03/2023, 17:28:57] [homebridge-unifi-protect] Child bridge process ended
[27/03/2023, 17:28:57] [homebridge-unifi-protect] Process Ended. Code: 1, Signal: null

[27/03/2023, 17:29:04] [homebridge-unifi-protect] Restarting Process...
[27/03/2023, 17:29:06] [homebridge-unifi-protect] Launched child bridge with PID 10060
[27/03/2023, 17:29:06] Registering platform 'homebridge-unifi-protect.UniFi Protect'
[27/03/2023, 17:29:06] [homebridge-unifi-protect] Loaded homebridge-unifi-protect v6.0.1 child bridge successfully
[27/03/2023, 17:29:06] Loaded 11 cached accessories from cachedAccessories.0EF8354E1149.
[27/03/2023, 17:29:07] Homebridge v1.6.0 (HAP v0.11.0) (homebridge-unifi-protect) is running on port 44494.
[27/03/2023, 17:29:07] [homebridge-unifi-protect] Connected to the UniFi Protect API at a.b.c.d.
TypeError: Cannot read properties of null (reading 'temperature')
    at ProtectNvrSystemInfo.getCpuTemp (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:209:47)
    at ProtectNvrSystemInfo.configureTemperatureSensor (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:201:94)
    at ProtectNvrSystemInfo.updateDevice (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:141:13)
    at ProtectNvrSystemInfo.configureAccessory (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:102:33)
    at new ProtectNvrSystemInfo (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:28:10)
    at ProtectNvr.ufpStartupHandler (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr.ts:184:23)
    at Object.onceWrapper (node:events:628:26)
    at ProtectApi.emit (node:events:513:28)
    at ProtectApi.bootstrapController (file:///usr/local/lib/node_modules/homebridge-unifi-protect/node_modules/unifi-protect/src/protect-api.ts:256:10)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
[27/03/2023, 17:29:08] [homebridge-unifi-protect] Child bridge process ended
[27/03/2023, 17:29:08] [homebridge-unifi-protect] Process Ended. Code: 1, Signal: null

[27/03/2023, 17:29:15] [homebridge-unifi-protect] Restarting Process...
[27/03/2023, 17:29:16] [homebridge-unifi-protect] Launched child bridge with PID 10076
[27/03/2023, 17:29:16] Registering platform 'homebridge-unifi-protect.UniFi Protect'
[27/03/2023, 17:29:16] [homebridge-unifi-protect] Loaded homebridge-unifi-protect v6.0.1 child bridge successfully
[27/03/2023, 17:29:16] Loaded 11 cached accessories from cachedAccessories.0EF8354E1149.
[27/03/2023, 17:29:17] Homebridge v1.6.0 (HAP v0.11.0) (homebridge-unifi-protect) is running on port 44494.
[27/03/2023, 17:29:18] [homebridge-unifi-protect] Connected to the UniFi Protect API at a.b.c.d.
TypeError: Cannot read properties of null (reading 'temperature')
    at ProtectNvrSystemInfo.getCpuTemp (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:209:47)
    at ProtectNvrSystemInfo.configureTemperatureSensor (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:201:94)
    at ProtectNvrSystemInfo.updateDevice (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:141:13)
    at ProtectNvrSystemInfo.configureAccessory (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:102:33)
    at new ProtectNvrSystemInfo (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr-systeminfo.ts:28:10)
    at ProtectNvr.ufpStartupHandler (file:///usr/local/lib/node_modules/homebridge-unifi-protect/src/protect-nvr.ts:184:23)
    at Object.onceWrapper (node:events:628:26)
    at ProtectApi.emit (node:events:513:28)
    at ProtectApi.bootstrapController (file:///usr/local/lib/node_modules/homebridge-unifi-protect/node_modules/unifi-protect/src/protect-api.ts:256:10)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
[27/03/2023, 17:29:18] [homebridge-unifi-protect] Child bridge process ended
[27/03/2023, 17:29:18] [homebridge-unifi-protect] Process Ended. Code: 1, Signal: null

[27/03/2023, 17:29:25] [homebridge-unifi-protect] Restarting Process...
[27/03/2023, 17:29:26] [homebridge-unifi-protect] Launched child bridge with PID 10097
[27/03/2023, 17:29:27] Registering platform 'homebridge-unifi-protect.UniFi Protect'
[27/03/2023, 17:29:27] [homebridge-unifi-protect] Loaded homebridge-unifi-protect v6.0.1 child bridge successfully
[27/03/2023, 17:29:27] Loaded 11 cached accessories from cachedAccessories.0EF8354E1149.
[27/03/2023, 17:29:27] Homebridge v1.6.0 (HAP v0.11.0) (homebridge-unifi-protect) is running on port 44494.
[27/03/2023, 17:29:31] [homebridge-unifi-protect] a.b.c.d: API error: Protect controller is taking too long to respond to a request. This error can usually be safely ignored.
[27/03/2023, 17:29:31] [homebridge-unifi-protect] a.b.c.d: API error: Original request was: https://a.b.c.d
```

**Comment by hjdhjd** [comment]
I think I know what this one is. Thanks again.

**Comment by hjdhjd** [comment]
Fixed in v6.0.2 Give it a try?

**Comment by jfmach** [comment]
@hjdhjd I confirm it seems fixed. I restarted the bridge a couple of times and it's not crashing. Thank you!
